tests/unit/modules/analytics.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+/* global describe, it */
+
+var chai = require('chai');
+var expect = chai.expect;
+
+var analytics = require('../../../static/js/modules/analytics');
+
+describe('analytics', function() {
+  it('sorts query parameters', function() {
+    var query = {
+      cycle: [2016, 2014],
+      sort: 'name',
```

Comment:
Nope. This is a test, and it's verifying that query parameters are sorted correctly--it doesn't matter what the values are.